iocore/net/quic/QUICCrypto.h

```diff
@@ -0,0 +1,121 @@
+/** @file
```

Comment:
QUIC use the TLS-1.3 to handshake and key deriving. But QUIC is not on TLS, because QUIC is on UDP:D So, this is really QUIC specific class, IMO.
The overview of QUIC and TLS 1.3 is in [Section 3 of draft-ietf-quic-tls-04](https://tools.ietf.org/html/draft-ietf-quic-tls-04#section-3)

For HTTP, current master branch can do 1-RTT handshake of TLS-1.3. But I'm not sure it can do 0-RTT handshake or not without any changes.